root/statistics/MusicBrainzHistory.js

```diff
@@ -8,17 +8,16 @@
  */
 
 import Layout from '../layout/index.js';
-import expand2react from '../static/scripts/common/i18n/expand2react.js';
 
 component MusicBrainzHistory(events: $ReadOnlyArray<StatisticsEventT>) {
   return (
-    <Layout fullWidth title={l_statistics('History')}>
-      <h1>{l_statistics('Our glorious history')}</h1>
+    <Layout fullWidth title={l_history('History')}>
+      <h1>{l_history('Our glorious history')}</h1>
       {events.length
         ? events.map((event) => {
-          const title = exp.l_statistics(
+          const title = exp.l_history(
             '{date} - {title}',
```

Comment:
Those are neither history event titles nor descriptions. Those should rather stay in the domain `statistics`.

Reply:
Not more than the editing form field title “Language” and the translation domain `language`.

Reply:
On second thoughts, after https://github.com/metabrainz/musicbrainz-server/pull/3312#discussion_r1662973188, if you want to mix event’s titles and descriptions with strings from the page layout, we should consider adding translation context keys to the event’s titles and descriptions.